components/shared/searchFilters/SearchFiltersColumn.tsx

```diff
@@ -44,48 +45,38 @@ const SearchFiltersColumn = ({
     <>
       <div
         key={facet.name}
-        className={cn(["relative", !isLast && "min-w-32 flex-1", isLast && "flex-2"])}>
-        <h3 className="mb-2 text-typo-caption uppercase">{getFacetTranslation(facetFilter)}</h3>
+        className={cn(["relative ml-[-4px]", !isLast && "min-w-32 flex-1", isLast && "flex-2"])}>
+        <h3 className="mb-2 pl-2 text-typo-caption uppercase">
+          {getFacetTranslation(facetFilter)}
+        </h3>
         <div
           className={cn([
-            "flex gap-1 text-typo-caption",
+            "flex gap-1 px-1 pt-2 text-typo-caption",
             !isLast && "flex-col",
-            isLast && "flex-row flex-wrap",
-            !isExpanded && "h-[98px] overflow-hidden",
+            isLast && "flex-row flex-wrap content-start",
+            !isExpanded && "h-[107px] overflow-hidden",
           ])}
           ref={elementRef}>
           {facet.values.map((value, index) => (
-            <button
+            <BadgeButton
               onClick={() => toggleFilter(facet.name, value.term, router)}
-              className={cn([
-                `h-[29px] w-auto self-start whitespace-nowrap rounded-full bg-background-overlay px-4 py-2
-                hover:animate-wiggle`,
-                searchParams.getAll(facet.name).includes(value.term) &&
-                  "bg-foreground text-background",
-              ])}
+              isActive={!!searchParams.getAll(facet.name).includes(value.term)}
               key={index}>
               {value.term}
-            </button>
+            </BadgeButton>
           ))}
         </div>
         {hasOverflow && (
-          <div
-            className="h-9 w-9 cursor-pointer"
+          <BadgeButton
+            classNames={cn(`pl-3 w-auto flex flex-row items-center self-start  ml-1`)}
             onClick={() => {
               setIsExpanded(prev => !prev)
             }}>
-            <button
-              className={cn(
-                `flex h-[29px] w-auto flex-row items-center self-start whitespace-nowrap rounded-full
-                bg-background-overlay pl-2 pr-4 text-typo-caption hover:animate-wiggle`,
-                isExpanded && "mt-1"
-              )}>
-              <Icon className={cn("h-8 w-8", isExpanded && "rotate-180")} name="arrow-down" />
-              <p>
-                {!isExpanded && "Flere"} {isExpanded && "Skjul"}
-              </p>
-            </button>
-          </div>
+            <Icon className={cn("h-8 w-8", isExpanded && "rotate-180")} name="arrow-down" />
```

Comment:
Im pretty sure "isExpanded" will end up in the class if you're using this time of if statement

```suggestion
            <Icon className={cn("h-8 w-8", isExpanded ? "rotate-180" : '')} name="arrow-down" />
```